ci_framework/roles/ci_setup/defaults/main.yml

```diff
@@ -17,4 +17,6 @@
 
 # All variables intended for modification should be placed in this file.
 # All variables within this role should have a prefix of "cifmw_ci_setup"
+
 cifmw_ci_setup_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_ci_setup_openshift_client_version: "4.13.0"
```

Comment:
Are we gonna stick to an specific 4.13.X version? Or maybe is a better idea to stick to the latest, stable, 4.13.Y? Seems like the mirror URL allows downloading the latest stable version for each version. This is not a must, I'm just asking if it's a better, as that will, while guaranteeing compatibility, have all the bugfixes of the cli tool.

Reply:
Good point. Let me switch to using stable instead of hard coding the version. We can switch to a version specific if we encounter issues later.